code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -137,7 +137,7 @@
 
 /obj/item/weapon/reagent_containers/pill/stox
 	name = "Sleeping pill"
-	desc = "Commonly used to treat insomnia."
+	desc = "Commonly used to treat insomnia. Contains 15 units of Soporific."

```

Comment:
How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.